X10D/src/Collections/DictionaryExtensions.cs

```diff
@@ -116,9 +263,9 @@ public static class DictionaryExtensions
         }
 #endif
 
-        if (dictionary.ContainsKey(key))
+        if (dictionary.TryGetValue(key, out var old))
```

Comment:
Same as above.  Since the assigned type here is not inherently apparent, explicitly declaring old as TValue is preferred.